css/style.css

```diff
@@ -714,6 +720,63 @@ textarea {
   .btn-contact {
     align-self: center;
   }
+
+  /* POPUP WINDOW DETAILS */
+
+  .popup-details {
+    padding: 1% !important;
+  }
+
+  .details-content {
+    width: 50%;
+    height: 95vh;
+    padding: 2% 1% !important;
+  }
+
+  .popup-img-container {
+    align-items: center;
+    margin-top: 0 !important;
+  }
+
+  .screenshot {
+    width: 100% !important;
+    height: 70% !important;
+  }
+
+  .popup-desc {
+    display: flex;
+    margin-top: 0 !important;
+    gap: 2%;
+  }
+
+  .popup-text-desc {
+    width: 70%;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+
+    /* color: #344563;
+    margin-bottom: 1%; */
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻